algorithms/C/maths/fibonacci-number/Makefile

```
@@ -0,0 +1,4 @@
+run: main.c
+	.\a.exe
+main.c:
+	gcc .\src\main.c
```

```suggestion
	gcc .\src\main.c  –lm
```

Not all gcc might be pre-configured to by default link to a proper math library. Explicitly linking the math library make sure the program is built correctly and the can find all the proper definitions that are being used in the program.

---

I will include it.